census/tract.py

```diff
@@ -32,11 +35,24 @@ def __init__(self):
             {'field': 'code', 'column': 'county_id', 'data': self.get_county_id}
         ]
 
-        self.cacheable_fields = ['fips']
+        self.cacheable_fields = ['fips','name','id']
```

Comment:
Do we need to cache by `id` also? What is using that?

Reply:
It's used in TractPopulation -> load_cache() to identify the name using the id. Which is then used for the skipping logic